AndroidSDK/src/com/leanplum/messagetemplates/MessageTemplates.java

```diff
@@ -90,18 +90,19 @@
 
     // Open URL.
     static final String DEFAULT_URL = "http://www.example.com";
-
+    static final String DEFAULT_BASE_URL_NEW = "https://leanplum/";
```

Comment:
I don't think we need to have both New and Old one. Can we make it an ActionContext argument and have "https://leanplum/" as the default one?

Reply:
IMO it's safer to have both. Just in case there is a bug somewhere else in the system, and maybe the new template isn't downloaded.

Reply:
In that case we should handle that in the bridge and not pollute the ActionContext arguments.